src/main/kotlin/racingcar/domain/Car.kt

```diff
@@ -0,0 +1,12 @@
+package racingcar.domain
+
+import racingcar.vo.Position
+
+class Car(position: Position = Position()) {
+    var position: Position = position
+        private set
+
+    fun move(canMove: () -> Boolean) {
+        position = if (canMove.invoke()) position.move() else position
+    }
+}
```

Comment:
```suggestion
class Car(val position: Position) {

    fun move(canMove: () -> Boolean) {
        if (canMove.invoke()) {
            position.move()
        }
    }
}
```

position 값에 대해 포장을 하셨군요! 그러면 차라리, position에 대한 값 변경을 position에게 역할을 주는게 어떨까요?

Reply:
position을 불변으로 만들어서 car에서 재할당을 해주는 방식으로 생각을 했는데,
position이 가변인 것이 더 가독성 측면에서 더 좋을 수 있겠네요